src/sdk/generatePassphraseWordList.js

```diff
@@ -0,0 +1,37 @@
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import path from 'path';
+
+// Get the directory path of the current module
+const currentModulePath = fileURLToPath(import.meta.url);
+const currentDirectory = path.dirname(currentModulePath);
+
+// Construct the absolute path to the passphraseWords.txt file
+const filePath = path.join(currentDirectory, 'passphraseWords.txt');
+const TopPassphrasePath = path.join(currentDirectory, 'topPassphrases.ts');
```

Comment:
You don't need to do all of this stuff with files. Just import the exported variable

Reply:
This whole file can be deleted

Reply:
see https://github.com/Tonomy-Foundation/Tonomy-ID-SDK/blob/master/src/sdk/util/passwords.ts#L2